### DISCUSSION TITLE
Account Hacked..

### DISCUSSION THREAD

**Discussion by sarthakdev143-lite** [community/community#132503] [Actions]
Hello GitHub Community,

I am reaching out here because I have been facing an issue with recovering my GitHub account after it was compromised. Over a month ago, I submitted a support ticket regarding this matter (#2852391), but unfortunately, I have not received any response yet.

Issue Details:
• My GitHub account's primary email address was changed by a hacker.
• I have been unable to regain access to my account through normal password reset procedures as I no longer have access to the associated email.

Steps Taken:
• Submitted a support ticket over a month ago (#2852391).
• Checked spam/junk folders regularly for any potential responses.
• Followed GitHub's account recovery guidelines and procedures to the best of my ability.

Request for Assistance:
• I urgently need help in recovering access to my GitHub account.
• Could someone from GitHub support team kindly assist in expediting my case or provide guidance on how to proceed further?
I appreciate any help or direction the community or GitHub staff can provide. Thank you in advance for your time and assistance.

Best regards,

@sarthakdev143-lite
Username of Account which got hacked : - @sarthakdev143

**Comment by Saulimedes** [comment]
I don't think the community can help here much. Try reaching the official support to maybe verify your identity and history of this incident.

**Reply by sarthakdev143-lite** [reply]
Bro.. I have registered a already registered ticket... but they are not replying..

https://github.com/user-attachments/assets/5e7d30f7-f4df-440f-b9dd-1e5aabe76c57

**Reply by Saulimedes** [reply]
I understand you might be angry what it isn't answered immediately. This is a horrible thing! But the community is unable to help you. When you submit a ticket it will be answered in order it was received based on the support team's resources and the amount of tickets ahead of yours. I know it might not be what you want to hear.
